src/harness/testcases/WINNF_FT_S_SSS_testcase.py

```diff
@@ -0,0 +1,206 @@
+  def test_WINNF_FT_S_SSS_7(self,config_filename):
+    """Corrupted certificate presented during registration.
+
+    Checks that SAS UUT response with fatal alert message.
+    """
+    config = loadConfig(config_filename)
+    self.assertTlsHandshakeFailure(client_cert=config['sasCert'],
+                                   client_key=config['sasKey'])
+
+  def generate_SSS_8_default_config(self, filename):
+    """Generates the WinnForum configuration for SSS_8"""
+    # Create the actual config for sas cert/key path
+
+    config = {
+      'sasCert': self.getCertFilename("self_signed_sas.cert"),
+      'sasKey': self.getCertFilename("sas.key")
+    }
+    writeConfig(filename, config)
+
+  @configurable_testcase(generate_SSS_8_default_config)
+  def test_WINNF_FT_S_SSS_8(self,config_filename):
+    """Self-signed certificate presented during registration.
+
+    Checks that SAS UUT response with fatal alert message.
+    """
+    config = loadConfig(config_filename)
+    self.assertTlsHandshakeFailure(client_cert=config['sasCert'],
+                                   client_key=config['sasKey'])
+
+  def generate_SSS_9_default_config(self, filename):
+    """Generates the WinnForum configuration for SSS_9"""
+    # Create the actual config for domain proxy cert/key path
+
+    config = {
+      'sasCert': self.getCertFilename("non_cbrs_signed_sas.cert"),
+      'sasKey': self.getCertFilename("non_cbrs_signed_sas.key")
+    }
+    writeConfig(filename, config)
+
+  @configurable_testcase(generate_SSS_9_default_config)
+  def test_WINNF_FT_S_SSS_9(self,config_filename):
+    """Non-CBRS trust root signed certificate presented during registration.
+
+    Checks that SAS UUT response with fatal alert message.
+    """
+    config = loadConfig(config_filename)
+    self.assertTlsHandshakeFailure(client_cert=config['sasCert'],
+                                   client_key=config['sasKey'])
+
+  def generate_SSS_10_default_config(self, filename):
+    """Generates the WinnForum configuration for SSS_10. """
+    # Create the actual config for domain proxy cert/key path
+
+    config = {
+      'sasCert': self.getCertFilename("wrong_type_sas.cert"),
+      'sasKey': self.getCertFilename("client.key"),
+      'sasThUrl':'localhost:5000'
+
+    }
+    writeConfig(filename, config)
+
+  @configurable_testcase(generate_SSS_10_default_config)
+  def test_WINNF_FT_S_SSS_10(self,config_filename):
+    """Certificate of wrong type presented during registration.
+
+    Checks that SAS UUT response with fatal alert message.
+    """
+    config = loadConfig(config_filename)
+    try:
+      self.assertTlsHandshakeFailure(client_cert=config['sasCert'],
+                                     client_key=config['sasKey'])
+    except AssertionError as e:
+      self.SasReset()
+      
+      # Initiate Full Activity Dump
+      self._sas_admin.TriggerFullActivityDump()
+      try:
+         self._sas.GetFullActivityDump()
```

Comment:
if missing code backported, you are still missing the `ssl_cert` and `ssl_key`parameters

Reply:
The test cases SSS.10 and SSS.15 have been removed. These test cases have dependency on another PR#296.